packages/appsync-modelgen-plugin/src/utils/process-connections.ts

```diff
@@ -41,11 +40,8 @@ export function makeConnectionAttributeName(type: string, field?: string) {
   // Make sure the logic gets update in that package
   return field ? camelCase([type, field, 'id'].join('_')) : camelCase([type, 'id'].join('_'));
```

Comment:
There could be cases where the generated field name matches any existing field names. Is this case considered?

Reply:
I'm not sure if the case is considered, I wasn't working on this section of code. If we want to try and create some accounting for that we can, it would probably make more sense to do this in a different PR though